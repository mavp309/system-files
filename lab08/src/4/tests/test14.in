3.14 2.72 0.00 0.00
