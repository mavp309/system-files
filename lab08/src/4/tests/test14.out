3.1400 2.7200
